assets/src/edit-story/components/helpCenter/useHelpCenter/index.js

```diff
@@ -140,6 +110,32 @@ const initial = {
   },
 };
 
+/**
+ * Turns a boolean map into a string key
+ *
+ * @param {Object} map an object to stringify
+ * @return {string} a key to provide to useEffect
+ */
+const createKeyFromBooleanMap = (map) => Object.keys(map || {}).join(' ');
+
+/**
+ * reforms the boolean map from the key returned by createKeyFromBooleanMap
+ *
+ * @param {string} key a string of space-separated map keys
+ * @return {Object} the map of the provided keys with with true as the value
+ */
+const createBooleanMapFromKey = (key) =>
+  key
+    .split(' ')
+    .sort((a, b) => a.localeCompare(b))
+    .reduce(
+      (accum, keyName) => ({
+        ...accum,
+        [keyName]: true,
+      }),
+      {}
+    );
```

Comment:
I feel like I just read this yesterday. Does this exist somewhere else too? If so, maybe we should centralize these functions.

Reply:
Yea it came from @embarks PR, I just moved it when I rebased so it's showing up as new code